corehq/apps/ota/views.py

```diff
@@ -59,7 +59,7 @@ def get_restore_response(domain, couch_user, app_id=None, since=None, version='1
                             status=401)
 
     project = Domain.get_by_name(domain)
-    app = Application.get_db().get(app_id) if app_id else None
+    app = Application.get(app_id) if app_id else None
```

Comment:
i think it depends on the use case `Application.get(app_id)` will get you a wrapped version of the app where as `Application.get_db().get(app_id)` will return a dictionary